engineering_tools/carma

```diff
@@ -249,6 +280,14 @@ carma-config__install() {
     fi
     docker run --name carma-config-tmp $IMAGE_NAME
 
+    # Extract the .env file content from the carma-config volume
```

Comment:
Also here, should allow passing the .env file as well

Reply:
here we actually shouldn't use passing env since the carma install function is intending to install version in the config. If they need to install different version, they have to download different config or download images individually.